cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v2/transactions.rst

```diff
@@ -0,0 +1,38 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _http-restful-api-transactions:
+
+====================================
+Transaction Service HTTP RESTful API
+====================================
+
+Some internals of the Transaction Service are exposed via HTTP REST endpoints.
+
+Size of Invalid Transactions
+----------------------------
+
+To get the number of invalid transactions in the system, issue an HTTP GET request::
+
+  GET <base-url>/transactions/invalid/size
+
+The response is a JSON string::
+
+  { "size": <size> }
+
+
+Truncate Invalid Transactions by time
+-------------------------------------
+
+To truncate invalid transactions before a specific time, issue an HTTP POST request::
+
+  POST <base-url>/transactions/invalid/remove/until
+
+with the timestamp in milliseconds as a JSON string in the body::
+
+  { "time" : <timestamp-ms> }
+
+**Note:** Since improperly removing invalid transactions may result in data inconsistency,
+please refer to <insert-link> for proper usage.
```

Comment:
Added a note here: https://github.com/caskdata/cdap/pull/1626

Should link to the page there.